bson/src/main/org/bson/codecs/pojo/Conventions.java

```diff
@@ -69,8 +69,17 @@ public final class Conventions {
      */
     public static final Convention USE_GETTERS_FOR_SETTERS = new ConventionUseGettersAsSettersImpl();
 
-
     /**
```

Comment:
Does this convention assume that the `CLASS_AND_PROPERTY_CONVENTION` (or equivalent) precedes it in the list of conventions?  It seems so, but if so perhaps it's worth stating, as it would seem reasonable to assume that this convention would discover all the fields and add them as properties rather than removing fields that have already been discovered by a previous convention.

Reply:
It doesn't require any other conventions but can be used in combination with them.